src/main/java/edu/cornell/kfs/paymentworks/service/PaymentWorksUploadSupplierService.java

```diff
@@ -24,9 +24,18 @@
 	String createSupplierUploadFile(List<PaymentWorksSupplierUploadDTO> paymentWorksSupplierUploadList,String directoryPath);
 	
 	/**
-	 * Deletes the file passed in.
-	 * @param fileName
+	 * Uploads approved suppkiers to PaymentWorks
 	 */
-	void deleteSupplierUploadFile(String fileName);
```

Comment:
I spoke with Purchasing about the deleting of upload files.  There are a couple of scenarios where a file might not get uploaded to payment works, so they opted to keep backups on our batch server.
I created another user story in the backlog to figure out how we want to clean up the upload files.  Sandy noticed that user story and she told me about the nightly job that deleting files older than ~90 (I think) days.  So we are going to make use of that.